packages/alea-frontend/pages/api/job-portal/get-organization-profile.ts

```diff
@@ -0,0 +1,20 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import { checkIfGetOrSetError, executeDontEndSet500OnError } from '../comment-utils';
+import { OrganizationData } from '@stex-react/api';
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+  if (!checkIfGetOrSetError(req, res)) return;
+  const id = req.query.id;
+  const results: OrganizationData[] = await executeDontEndSet500OnError(
+    `SELECT id,companyName,incorporationYear,isStartup, about, companyType,officeAddress,officePostalCode,website,domain
+    FROM organizationProfile 
+    WHERE id = ? 
+    `,
+    [id],
+    res
+  );
+  if (!results || !results.length) {
+    return res.status(200).json([]);
+  }
+  res.status(200).json(results[0]);
```

Comment:
You can simplify the code but why are you returning `[]` ? 

Secondly, if `!results` is true, you should return early. Otherwise, there will be two `res.status` calls - which will results in hard-to-debug errors
```suggestion
  res.status(200).json(results[0] || []);
```